### DISCUSSION TITLE
How to install the gurobi solver?

### DISCUSSION THREAD

**Discussion by mapr7** [ZEN-universe/ZEN-garden#98] [Installation & Setup] [answered]
How do I install the Gurobi software and license needed to run the optimization?

**Comment by mapr7** [comment, marked as answer]
https://support.gurobi.com/hc/en-us/articles/360044290292-How-do-I-install-Gurobi-for-Python-

**Reply by johburger** [reply]
In addition to the package installation, here is documentation regarding the academic gurobi license:
https://www.gurobi.com/academia/academic-program-and-licenses/

**Comment by jacob-mannhardt** [comment]
1. Register and request a free academic license: https://www.gurobi.com/downloads/gurobi-software/
2. Follow instructions on how to install the license:
- open command/terminal prompt
- copy paste the your grbgetkey and hit enter. Make sure your computer has internet access during the installation.
